config/requests.yml.tmpl

```diff
@@ -0,0 +1,14 @@
+# requests.yml
+
+defaults: &defaults
+  umlaut_base: https://getit.princeton.edu
+  bibdata_base: http://bibdata.princeton.edu/bibliographic
```

Comment:
That is what I was thinking so we have a clean way to point at test versions of external services like bibdata down the road. I think we can  expose config values to the javascript layer directly.